src/components/pollutedLocations/PollutedCardList.tsx

```diff
@@ -0,0 +1,18 @@
+import PollutedLocation from "../../types/PollutedLocation";
+import PollutedLocationCard from "./PollutedLocationCard";
+
+const PollutedCardList = (list: PollutedLocation[]) => {
```

Comment:
`PollutedCardList` should have an `interface Props` and be of type `React.FC<Props>`. You can check [`Map`](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/staging/src/components/map/Map.tsx#L11) for example